File: back/src/main/java/com/superlucky/model/Prize.java
package com.superlucky.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.Data;

@Entity
@Data
public class Prize {
    @Id
    @GeneratedValue(strategy=GenerationType.IDENTITY)
    public Long id;
    public String title;
    public String description;
    public String imagePath;
}

File: back/src/Entity/Prize.ts
import { Entity, PrimaryGeneratedColumn, Column } from "typeorm";

@Entity()
export default class Prize {
    constructor() {
        this.name = "";
    }
    @PrimaryGeneratedColumn()
    id?: number;

    @Column()
    title: string;

    @Column()
    description: string

    @Column()
    imagePath: string
}
